### DISCUSSION TITLE
It will work only with API?

### DISCUSSION THREAD

**Discussion by devaniljr** [pezzolabs/pezzo#274] [Q&A]
I want to integrate with my Ruby on Rails app, but since there is no sdk client for it, I wonder if using API will work. And how this can be done.

**Comment by arielweinberger** [comment]
Hi @devaniljr absolutely - the API spec can be found in the docs:
https://docs.pezzo.ai/api-reference/health/performs-a-health-check

**Reply by FernandoTWE** [reply]
I would also like to be able to make POST requests to get answers directly via API, but I don't know if this is possible.

**Reply by arielweinberger** [reply]
@FernandoTWE That won't be possible. Pezzo is only a monitoring solution, we don't handle the request to the AI model itself.
